app/notify/notify_email/candidate_signin_link.5b451894-3640-47df-a119-6461ecb890d9.md

```diff
@@ -4,4 +4,4 @@ Click the link below to sign in to your school experience dashboard
 
 ((confirmation_link))
 
-This link will expire within 1 hour.
```

Comment:
Ah - this is wrong in the email template - it is actually 1 hour, can you pull this line out and amend the template in production, thanks